DashWallet/Sources/UI/Home/Protocols/DWHomeProtocol.h

```diff
@@ -42,7 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)homeModelWantToReloadShortcuts:(id<DWHomeProtocol>)model;
 @end
 
-@protocol DWHomeProtocol <DWTxDisplayModeProtocol>
+@protocol DWHomeProtocol <DWTxDisplayModeProtocol, DWDashPayReadyProtocol>
```

Comment:
Contacts VC requires `DWDashPayReadyProtocol` which was implemented by the home model before. Perhaps there is a path to refactor this in the future stories.